tests/end2end/data/downloads/mhtml/complex/complex.html

```diff
@@ -22,7 +22,7 @@
         
         <!-- don't parse non-CSS styles -->
         <style rel="stylesheet" type="text/qss">
-        @import "actually-it's-css";
```

Comment:
Yeah, at least it attempts to download it, which caused the tests to fail because of a 404. Can't do much about it, but I changed the name and added the empty file to make the tests pass.